salesforce/events.go

```diff
@@ -258,8 +258,6 @@ func (c *Connector) postToSFAPI(ctx context.Context, body any, path string, enti
 		return nil, err
 	}
 
-	fmt.Println("location: ", location)
```

Comment:
did yo accidentally commit this?

Reply:
No, this was a linter warning, unless we need this?